src/emulator/modules/CMakeLists.txt

```diff
@@ -259,3 +259,5 @@ SceVideodec
 SceVoice
 SceVoiceQoS
 SceVshBridge)
+
+target_link_libraries(modules INTERFACE util spdlog)
```

Comment:
This can be reverted.

`modules` is just an `INTERFACE` library that groups the existing module libraries into one for easier dependency management, it doesn't need any dependencies itself.

This file is also automatically generated, so we shouldn't edit it without editing `gen-modules` too.

Reply:
thank god it's autogenerated